idx/cassandra/cassandra.go

```diff
@@ -266,8 +266,14 @@ func (c *CasIdx) AddOrUpdate(data *schema.MetricData, partition int32) {
 	}
 
 	if updateIdx {
+		// set the new metricdef based on the data, however if this data will go back in time
+		// it'll be rejected data, and we should stick to the higher LastUpdate we already had.
 		def := schema.MetricDefinitionFromMetricData(data)
+		if inMemory && existing.LastUpdate > def.LastUpdate {
```

Comment:
- metrictank writes update to cassandra with LastUpdate = 10
- metrictank is restarted
- metrictank loads the index from cassandra, lastUpdate=10
- metrictank starts ingesting metrics from 6hours ago.
- first point seen has Time=0, so MetricDefinitionFromMetricData(data) will set LastUpdate to 0.
- existing.LastUpdate=10, def.LastUpdate=0.      10 > 0